frontend.conf

```diff
@@ -1,3 +1,16 @@
+# -----------------------------------------------------------------------------#
```

Comment:
In general, the current style does not imply the use of any ascii-frames. I'm not sure if this improves the readability of the config files, so I would remove it.